src/h3/unstable/v4.py

```diff
@@ -0,0 +1,135 @@
+# flake8: noqa
+
+# todo: what are the pythonic names?
+
+# what if we put this in an object? what would it look like?
+# a temporary API wrapper
+
+# what if we organized by module
+
+from ..api.basic_str import (
+    compact as
+        compact,
+
+    edge_length as
+        hex_edge_length_avg,
+
+    geo_to_h3 as
+        point_to_cell,
+
+    get_destination_h3_index_from_unidirectional_edge as
+        edge_destination,
+
+    get_h3_indexes_from_unidirectional_edge as
+        directed_edge_cells,
```

Comment:
A little inconsistent that this doesn't have "to" in the name but `cells_to_directed_edge` does. I think I understand why you chose this but it does jump out at me when reading the code.